packages/forms/src/Components/CodeEditor.php

```diff
@@ -12,4 +14,18 @@ class CodeEditor extends Field
      * @var view-string
      */
     protected string $view = 'filament-forms::components.code-editor';
+
+    protected Language | Closure $language = Language::PlainText;
```

Comment:
Instead of needing a `PlainText` case, I think that should just be represented by `null` language.